samples/snippets/src/main/java/aiplatform/CreateTrainingPipelineImageClassificationSample.java

```diff
@@ -204,25 +200,6 @@ static void createTrainingPipelineImageClassificationSample(
         System.out.format("Deployed Model Id: %s\n", deployedModelRef.getDeployedModelId());
       }
 
-      ExplanationSpec explanationSpec = modelResponse.getExplanationSpec();
-      System.out.println("Explanation Spec");
-
-      ExplanationParameters explanationParameters = explanationSpec.getParameters();
-      System.out.println("Parameters");
-
-      SampledShapleyAttribution sampledShapleyAttribution =
-          explanationParameters.getSampledShapleyAttribution();
-      System.out.println("Sampled Shapley Attribution");
-      System.out.format("Path Count: %s\n", sampledShapleyAttribution.getPathCount());
-
-      ExplanationMetadata explanationMetadata = explanationSpec.getMetadata();
-      System.out.println("Metadata");
-      System.out.format("Inputs: %s\n", explanationMetadata.getInputsMap());
-      System.out.format("Outputs: %s\n", explanationMetadata.getOutputsMap());
-      System.out.format(
-          "Feature Attributions Schema_uri: %s\n",
-          explanationMetadata.getFeatureAttributionsSchemaUri());
-
       Status status = trainingPipelineResponse.getError();
       System.out.println("Error");
```

Comment:
Is this suppose to error or does this need a catch stateement?

Reply:
No, the service might run into an error during model training that is unrelated to the request sent by the customer. There isn't an error that we can catch for this. (Same as other samples.)